docs/Joining-Data.md

```diff
@@ -8,9 +8,8 @@
 # Summary
 
 Similar to the well-known [SQL construct](<https://en.wikipedia.org/wiki/Join_(SQL)>),
-the Zed language provides a `join` operator that can be used to combine data
-sources. This cookbook describes how it can be used with the Brim application
-and discusses its limitations.
+the Zed language provides a [`join`](https://zed.brimdata.io/docs/language/operators/join/) operator that can be used to combine data
+sources. This cookbook describes how it can be used with the Brim application.
```

Comment:
Since multiple pools can now be referenced in a Zed query in Brim, a case could have been made to drop this Join article altogether. However, since the multi-line editor is not yet here, and app-heavy users may not have yet read the Zed docs closely yet, I figured a few targeted refs to help the user successfully run through the Zed language Join tutorial might still be worthwhile. Maybe a few users have it bookmarked, right?